dluhc-component-library/src/api/planningData/api.ts

```diff
@@ -16,9 +17,17 @@ export const fetchDataset = async (geometry: Geometry) => {
   });
 };
 
-export const fetchDatasetList: () => Promise<DatasetResponse> = async () =>
+export const fetchDatasets: () => Promise<DatasetResponse> = async () =>
```

Comment:
Is this used anywhere but inside the userQuery below? If not i think we can pull it into there and use the reactQuery typings to help us

Reply:
Yeah it's only used the query, I've moved it into there. They type of the query is now `useQuery<DatasetResponse, unknown, Dataset[]>`. Is `unknown` okay for the error type or is there something more specific that could work?